controllers/mysql_combined_test.go

```diff
@@ -152,24 +157,31 @@ func RunMySQLUserHappyPath(ctx context.Context, t *testing.T, mySQLServerName st
 		ctx,
 		mysql.MySQLDriverName,
 		fullServerName,
-		mySQLDBName,
+		"mysql",
 		mysql.MySQLServerPort,
 		adminUser,
 		adminPassword)
 	assert.NoError(err)
 
+	expectedRoles := mysql.SliceToSet(updatedRoles)
```

Comment:
May be lacking some tests around the the "DBName doesn't exist" case - it was explicitly mentioned in one of the comments but not sure that this test covers it?